RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -500,9 +717,134 @@ HcalNoiseInfoProducer::filldigis(edm::Event& iEvent, const edm::EventSetup& iSet
 	      }
           } // end of HBHE check
      } // loop on HcalCalibDigiCollection
+
+     // now match the laser monitor data by fiber (in time) 
+     if( fillLaserMonitor_ ) {
+       // check for any fibers without data and fill
+       // them so we dont run into problems later
+       for( unsigned idx = 0; idx < laserMonDetTypeList_.size(); ++idx ) {
+           if( lasmon_adcs[idx].empty() ) {
+               for( int i = 0; i < 10; ++i ) {
```

Comment:
this can be written as lasmon_adcs[idx]=std::vector<int>(10,-1)

Reply:
yes, done